clustered/client/src/main/java/org/ehcache/clustered/client/internal/service/DefaultClusteringService.java

```diff
@@ -383,23 +383,17 @@ protected boolean isStarted() {
     try {
       if (configuration.isAutoCreate()) {
         try {
-          this.entity.validateCache(cacheId, clientStoreConfiguration);
-        } catch (ClusteredTierValidationException ex) {
-          if (ex.getCause() instanceof InvalidStoreException) {
-            try {
-              this.entity.createCache(cacheId, clientStoreConfiguration);
-            } catch (TimeoutException e) {
-              throw new CachePersistenceException("Unable to create clustered tier proxy '"
-                  + cacheIdentifier.getId() + "' for entity '" + entityIdentifier
-                  + "'; create operation timed out", e);
-            }
-          } else {
-            throw ex;
+          entity.createCache(cacheId, clientStoreConfiguration);
+        } catch (ClusteredTierCreationException e) {
+          // An InvalidStoreException means the cache already exists. That's fine, the validateCache will then work
+          if (!(e.getCause() instanceof InvalidStoreException)) {
+            throw e;
           }
         }
-      } else {
-        this.entity.validateCache(cacheId, clientStoreConfiguration);
       }
+
+      entity.validateCache(cacheId, clientStoreConfiguration);
```

Comment:
The idea behind that is the following use-cases
* No cache exists. The client creates one and validate it
* A cache exists. The client receives a creation exception and then validate successfully
* No cache exist but one is in creation. So the create will fail and the validate will be successful
When it is the other way around, you get
* No cache exists. The client fails to validate and creates one
* A cache exists, the validate works
* No cache exists but one is in creation. The validate fails so the create occurs and fails